pkgs/development/compilers/ocaml/4.12.nix

```diff
@@ -1,7 +1,7 @@
 import ./generic.nix {
   major_version = "4";
   minor_version = "12";
-  patch_version = "0-beta1";
+  patch_version = "0";
   src = fetchTarball {
```

Comment:
Please remove the `src` attribute and add a `sha256` attribute.

Reply:
done in the squashed commit